pkg/network/network.go

```diff
@@ -25,7 +26,13 @@ var templates embed.FS
 // ReconcileNetwork reconciles the network settings for a firewall
 // in the current stage it only changes the FRR-Configuration when network prefixes or FRR template changes
 func ReconcileNetwork(f firewallv1.Firewall, log logr.Logger) (bool, error) {
-	kb := netconf.NewKnowledgeBase(MetalKnowledgeBase)
+	// FIXME use zapr ?
```

Comment:
I'm totally agree with this. But `zapr` just wraps `zap` logger into `logr.Logger` interface. So we would have to change the `config` struct on the `metal-networker` side.